AngularJS/src/index.pug

```diff
@@ -0,0 +1,35 @@
+doctype html
+html(lang='en')
+  head
+    title WBooks
+    meta(charset='UTF-8')
+    meta(name='viewport'
+      content='width=device-width, initial-scale=1')
+    link(href='/css/vendor.css' rel='stylesheet')
+    link(href='/css/application.css' rel='stylesheet')
+    base(href='/')
+    script(src="https://ajax.googleapis.com/ajax/libs/angularjs/1.7.8/angular.min.js")
+  body(ng-app="wbooks")
+    nav(class="nav")
+      img(class="nav-logo", src="assets/wbooks_logo.svg", alt="Wolox")
+      div(class="nav-items")
+        img(class="nav-item", src="assets/notifications.svg", alt="notifications")
+        img(class="nav-item", src="assets/add_book.svg", alt="add_book")
+        img(class="nav-avatar", src="assets/avatar.png", alt="avatar")
+    div(class="container", ng-controller="dashboardCtrl as dbc")
```

Comment:
Nothing of this should be in here, only the ui-content. Move all of the dashboardCtrl to the controller only